src/main/scala/ru/d4team/chat/services/ChatRoomsService.scala

```diff
@@ -42,4 +43,11 @@ final case class ChatRoomsServiceMapImpl(private[services] val rooms: ChatRoomsM
         case None       => None // TODO (Aleksei Litkovetc) - Log a leaving from non-existent room
       }
     }.unit
+
+  def broadcast(channel: Channel[WebSocketFrame], room: String, message: String): Task[Unit] =
+    rooms.get(room).fold[Task[Unit]](ZIO.unit) { room =>
+      ZIO.foreachParDiscard(room.view.filterKeys(_ != channel.id).values) { channel =>
```

Comment:
Let's move it to val like this: 
```
      val otherChannels = room.view.filterKeys(_ != channel.id).values
       ZIO.foreachParDiscard(otherChannels) { channel => ...
```
maybe otherChannels isn't accurate enough and you have own variant

Reply:
Done